cmd/run.go

```diff
@@ -82,6 +82,11 @@ func SetupRunFlags(cmd *cobra.Command) {
 		- hypersync: Will sync by downloading historical state, and will NOT
 		  download historical blocks. Can only be set if HyperSync is true.`)
 
+	// Mempool
+	cmd.PersistentFlags().Uint64("batch-persist-frequency-milliseconds", 30000,
```

Comment:
Why not call this the same as the argument? `mempool-backup-interval-milliseconds`